src/test/java/core/basesyntax/service/implementation/ValidatorImplTest.java

```diff
@@ -38,19 +38,13 @@ public void validate_invalidFruitName_notOk() {
     }
 
     @Test
-    public void validate_invalidFruitQuantity_notOk() {
-        boolean actual = validator.validate("b,banana,not");
-        Assert.assertFalse("Invalid quantity", actual);
-    }
```

Comment:
move this test back and add regex to validator
you didn't fix all the comments - check conversations above